Example/HorizonCalendarExample/HorizonCalendarExample/Demo View Controllers/SingleDaySelectionFooterDemoViewController.swift

```diff
@@ -0,0 +1,92 @@
+//  Created by Sebastien Audeon on 11/26/20.
+//  Copyright © 2020 Airbnb. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import HorizonCalendar
+
+class SingleDaySelectionFooterDemoViewController: DemoViewController {
```

Comment:
```suggestion
final class MonthFootersDemoViewController: DemoViewController {
```

I'd prefer to keep the demos as focused as possible, rather than combining independent concepts like single day selection and month footer configuration. Could we make the following changes (in addition to the naming suggestion above):

1. Remove all selection related naming / code
2. Give the footer some more interesting content, rather than just repeating the date in YYYY-MM format. Curious if you have any ideas / inspiration from your own projects of what would be useful in a footer?
3. Increase the value of `monthDayInsets.bottom` so that the footer isn't right up against the last week row of the month
4. Update the indentation to be 2 spaces instead of 4

Reply:
1. ✅ 
2. I haven't found the need just yet to add a footer to these in my own projects, but I can definitely make an attempt at coming up with something...
3. ✅ 
4. ✅